nginx/tests/test_nginx.py

```diff
@@ -1,11 +1,12 @@
 import requests
+from testcontainers.core.waiting_utils import wait_for_logs
 
 from testcontainers.nginx import NginxContainer
 
 
 def test_docker_run_nginx():
-    nginx_container = NginxContainer("nginx:1.13.8")
-    with nginx_container as nginx:
+    with NginxContainer("nginx:1.24.0") as nginx:
+        wait_for_logs(nginx, 'ready for start up')
```

Comment:
Great! Is this something we should move into the container implementation?

Reply:
Moved logic to be part of the `start` method 👍